### PR TITLE
*: support the statement of "rename database"

### PR DESCRIPTION
**Pull request by zimulala**
### What problem does this PR solve?
Support the statement of "rename database".
`RENAME {DATABASE | SCHEMA} db_name TO new_db_name;`
Related to [pr](https://github.com/bb7133/parser/pull/2).
The related MySQL [docs](http://download.nust.na/pub6/mysql/doc/refman/5.1/en/rename-database.html).

### What is changed and how it works?
Support the syntax of "rename database".

### Check List 

Tests 

 - Unit test

Code changes

 - Has interface methods change

### CODE REVIEW

**Comment by zimulala** [comment]
WIP：Becase the parser version needs to be updated.

**Review by Deardrops — COMMENTED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -74,3 +74,5 @@ require (
 )
 
 go 1.13
+
+replace github.com/pingcap/parser => github.com/bb7133/parser v0.0.0-20200429032743-4494e5f18a8d
```
Comment (bb7133) [inline go.mod:78]: I think you can avoid `replace` by using `go get`?

**Review by crazycs520 — COMMENTED** [review]
LGTM

File: go.mod (hunk 1 above)
Comment (zimulala) [reply go.mod:78]: It needs to replace the import packages. Maybe we needn't to it.

Comment (bb7133) [reply go.mod:78]: Sure, thanks.

**Pull request merged**